doc/api/deprecations.md

```diff
@@ -3303,9 +3303,13 @@ changes:
     - v19.2.0
     pr-url: https://github.com/nodejs/node/pull/45576
     description: Documentation-only deprecation.
+  - version:
+    - REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/45526
+    description: Runtime deprecation.
```

Comment:
Wrong order. BTW we don't need a list for the `version`, as it's a semver major it won't be backported (but also a list is fine).
```suggestion
    - REPLACEME
    pr-url: https://github.com/nodejs/node/pull/45526
    description: Runtime deprecation.
  - version: 
    - v19.2.0
    pr-url: https://github.com/nodejs/node/pull/45576
    description: Documentation-only deprecation.
```